content/services/interface.md

```diff
@@ -18,3 +18,47 @@ The procedure of the Notification-Event Translation System, works as follows,
 1. Upon creation of the Interface, and set up of a forge repository to look for, the interface subscribes to the forge's notifications.
 2. Every notification received by the interface from the forges, is then identified and translated into it's respective `Event` model, be it an Issue or PR.
 3. Upon conversion into the respective model, Interface processes these `Events`, and converts them into the model required for the destination forge.
+
+## Setting up the Development Environment
+
+In order to test out and utilise the Northstar lookup service, we will need to set
+up an interface to be run on the local machine. There are a few configuration 
+changes that must be made for the interface to be recognized by the lookup service.
+
+Changes to the `config/settings.toml`,
+```toml
+[default.system]
+northstar = "http://computer.domain.com:port"
```

Comment:
Can you please mention that the hostname can be obtained using

```bash
hostname --fqdn
```

Reply:
Changes have been made, please check if that would work or if the text must be changed.